### DISCUSSION TITLE
Next 13 - Dynamic Routes Error

### DISCUSSION THREAD

**Discussion by hizuvie** [vercel/community#2003] [Help] [answered]
### Summary

Hi everyone, I'm having a problem

I'm implementing dynamic routes to make a product page in the form /products/PRODUCT-HANDLE but I'm not sure why it's like that, till now i spent 20 hours to solve it but still not working, can anyone help me..

my folder
`app/products/[slug]/page.js`

my page.js content
`export default function Product() {
  return <div>Product</div>
}`

I'm getting a 404 error, and I check the console and it doesn't give me any error information

**Comment by hizuvie** [comment]
Hello everyone, this error is currently not fixed, can anyone see the cause of it? I am very depressed...

**Comment by amyegan** [comment]
Hi @hieupv4 👋 Does it work when you run it on your local environment, or is the 404 happening in all environments?

**Comment by hizuvie** [comment, marked as answer]
Thanks everyone, problem solved, error is because i run latest node version, 16.xx is needed
